src/audio/kpb.c

```diff
@@ -157,7 +157,7 @@ static void kpb_lock_init(struct comp_data *kpb)
 
 #endif /* __ZEPHYR__ */
 
-/**
+/*
```

Comment:
Why would this need to be reverted if we build doxygen in C files ?

Reply:
Doxygen looks at `/**` comments only, it ignores single `/*`.